server/api/workspace/workspace.controller.ts

```diff
@@ -172,11 +172,19 @@ class WorkspaceController {
 
     const updatedWorkspace = await workspace.save();
 
-    res.json(updatedWorkspace);
+    res.json(sanitizeWorkspace(updatedWorkspace));
   }
 
 }
 
+// Don't return kibana saved objects from workspace endpoints
+function sanitizeWorkspace(workspace: Workspace) {
```

Comment:
Do you think we should just exclude the kibana saved objects by default in every query using this approach?

https://typeorm.io/#/select-query-builder/hidden-columns